01_fib/sriyapaka/main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+
+	fib(7) //Provide the input, programmed for both Positive and Negative Series
+}
+
+func abs(x int) int {
+
+	if x < 0 {
+		return -x
+	}
+	return x
+}
+
+func fib(n int) {
+
+	var a int //first term
+	var b = 1 //second term
+	var c int //next term
+	var absn = abs(n)
+	var sign = n / absn
+
+	for i := 0; i < absn; i++ {
+		if i > 0 {
```

Comment:
It's functionally correct, but could be improved:
Don't use conditions on loop index but rearrange your loop.
Here is what _we_ have come up with:
```go
	sign := // 1 for positive n ; -1 for negative n
	n1, n2 := 0, 1
	for j := 0; j < n*sign; j++ {
		n1, n2 = n2, n1+sign*n2
		// print n1
	}
```